### PR TITLE
Fix error in api review

### PR DESCRIPTION
**Pull request by chenrujun**
Fix error in api review: https://apiview.dev/Assemblies/Review/2ebd7c87eaf047aaa9f13359a6fd38b2#
New api review (after commits in this PR): https://apiview.dev/Assemblies/Review/32fa8bce24ef4edeb3769dbb3d529a0c

Current package structure: 
image

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @JonathanGiles .
Please help to review this PR.

Relates api-reviews:
1. original one: https://apiview.dev/Assemblies/Review/2ebd7c87eaf047aaa9f13359a6fd38b2#
2. new one:https://apiview.dev/Assemblies/Review/32fa8bce24ef4edeb3769dbb3d529a0c

**Comment by chenrujun** [comment]
Hi, @zhichengliu12581 , please help to review this PR.

File: sdk/keyvault/azure-security-keyvault-jca/src/main/resources/module-info.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * 2. When used by other java 9 modules,
+ *    module-info.java must NOT have content like "requires org.apache.httpcomponents.httpclient;".
+ *
+ * To achieve this, we need these steps:
+ * 1. Avoid compile error in other module by deleting contents like "requires org.apache.httpcomponents.httpclient;".
+ * 2. Avoid compile error in current module by moving module-info.java out of src/main/java/.
+ * 3. Package module-info.class into xxx.jar by configuring moditect-maven-plugin.
+ * 4. Package module-info.java into xxx-source.jar by putting module-info.java in src/main/resources/.
+ * 5. Exclude module-info.java into xxx.jar by configuring maven-jar-plugin.
+ */
```
Comment (chenrujun) [inline sdk/keyvault/azure-security-keyvault-jca/src/main/resources/module-info.java:18]: @michaelqi793 , FYI.

Comment (michaelqi793) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/resources/module-info.java:18]: LGTM

**Comment by zhichengliu12581** [comment]
LGTM

**Comment by michaelqi793** [comment]
LGTM!

**Review by backwind1233 — APPROVED** [review]
LGTM

**Pull request merged**